docs/java/release-notes.md

```diff
@@ -15,10 +15,14 @@ import MvnBadge from '../../src/sap/sdk-java/MvnBadge'
 
 <MvnBadge />
 
+:::danger
+SAP Cloud SDK for Java V2 is in maintenance. To continue using the latest features, please, migrate to SDK V3.
```

Comment:
I like the increase in severity of this node 😄

I would recommend to take over the link to our migration guide though 😉